compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1930,6 +1930,44 @@ impl Arch {
     }
 }
 
+crate::target_spec_enum! {
+    pub enum Vendor {
+        Amd = "amd",
+        Apple = "apple",
+        Espressif = "espressif",
+        Fortanix = "fortanix",
+        Ibm = "ibm",
+        Kmc = "kmc",
+        Mti = "mti",
+        Nintendo = "nintendo",
+        Nvidia = "nvidia",
+        Openwrt = "openwrt",
+        Pc = "pc",
+        Risc0 = "risc0",
+        Sony = "sony",
+        Sun = "sun",
+        Unikraft = "unikraft",
+        Uwp = "uwp",
+        Vex = "vex",
+        Win7 = "win7",
```

Comment:
Any idea what "win7" is? Makes me think of Windows 7, which is a weird name for a vendor.

Reply:
Ugh, ok. Thanks for the info!